app/components/BigBtn1/tests/index.test.js

```diff
@@ -0,0 +1,10 @@
+// import React from 'react';
+// import { shallow } from 'enzyme';
+
+// import BigBtn1 from '../index';
+
+describe('<BigBtn1 />', () => {
+  it('Expect to have unit tests specified', () => {
```

Comment:
I will also post once by the tests I think we should write some before we go onto to other work or merge to master.